### PR TITLE
blog: announce poetry 1.8.0

### PR DESCRIPTION
**Pull request by radoering**

### CODE REVIEW

File: content/blog/2024-02-18-announcing-poetry-1-8-0.md [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+source. This is the last deprecation in a series of changes to simplify and improve
+Poetry's handling of package sources.
+
+### Upgrading warning about inconsistent lockfile to an error
+
+In the past, Poetry only printed a warning when running `poetry install` and the lockfile
+was inconsistent with the pyproject.toml. This happened if dependencies were changed
+in the pyproject.toml without running `poetry lock --no-update`.
+As a result, `poetry install` might fail or succeed with expected or unexpected results.
+This warning has been upgraded to an error to avoid issues that are hard to debug.
```
Comment (ulgens) [inline content/blog/2024-02-18-announcing-poetry-1-8-0.md:88]: Is there a better way to explain this change? I read it multiple times, but I still have no idea about what the change is.

Comment (radoering) [reply content/blog/2024-02-18-announcing-poetry-1-8-0.md:88]: There is certainly always room for improvement but I don't know how to explain it better. A warning becomes an error. In other words, the command will fail with return code unequal zero. If you have never seen the mentioned warning, it is probably irrelevant to you.

Comment (ulgens) [reply content/blog/2024-02-18-announcing-poetry-1-8-0.md:88]: I believe everyone using poetry saw the warning multiple times 🙂 I'm not sure why, but I think that "upgrade" comes with other changes, and the text was missing it. After rereading it, it feels more clear now. Let me share my other confusions, and the jury can decide if they need any updates:
* I'm missing how `--no-update` parameter is relevant. I checked the PR and the discussions but couldn't find any reference. The only thing I saw was `"Run `poetry lock [--no-update]` to fix it.` on an unedited line, which I believe makes better sense, emphasizing that it's just an optional parameter.
* `This happened if dependencies were changed in the pyproject.toml without running` This only covers a small portion of the relevant cases shared by people on the PR. My understanding is the issue and the solution is more about the hash from lock file not matching the relevant fields of the `pyproject.toml` file, not which file was edited by whom or for which purpose.

Comment (radoering) [reply content/blog/2024-02-18-announcing-poetry-1-8-0.md:87]: I rephrased the paragraph also considering the changed wording from python-poetry/poetry#8962. I hope it is clearer now. I still believe that changing something from warning to error can be called an upgrade but I am open to better wording. 😄

**Review by abn — APPROVED** [review]

**Pull request merged**